### DISCUSSION TITLE
Installation of v1.11.4

### DISCUSSION THREAD

**Discussion by choonho** [cloudforet-io/community#122] [Q&A] [answered]
Question about Installation v.1.11.4

https://github.com/cloudforet-io/charts#cloudforet-helm-charts

**Comment by whdalsrnt** [comment, marked as answer]
Hi, @choonho.

How can I help you?

**Reply by choonho** [reply]
I just want to add this discussion link in the installation document.
So people who has trouble in installation can easily understand known problems during installation.

**Comment by choonho** [comment]
# Use different namespaces.

If I want to change k8s namespaces instead of **spaceone** and **spaceone-plugin**.

What is changed values?

**Reply by whdalsrnt** [reply]
Hi, @choonho.

Refer to the link below for how to change the namespace.

https://github.com/cloudforet-io/charts/blob/master/examples/values/change_namespace_values.yaml

Thank you.
